arch/x86/src/start.rs

```diff
@@ -15,8 +15,8 @@ pub unsafe extern "C" fn start() {
     lea     ebp, _estack
 
     # Zero out the .bss section
-    mov     eax, _szero
-    mov     ebx, _ezero
+    lea     eax, _szero
```

Comment:
```suggestion
    lea     eax, [_szero]
```

Reply:
I don't agree that this makes it clearer for me. Using `lea` without square brackets seems common, and is how I learned to write x86 assembly. But that's not a hill I'll die on.

Reply:
I'm also a bit more accustomed to seeing an explicit memory expression being passed as an operand to `lea`, so I'd personally second Branden's suggestion.